addons/l10n_br_website_sale/static/tests/tours/brazilian_address.js

```diff
@@ -54,6 +54,14 @@ registry.category("web_tour.tours").add("test_brazilian_address", {
             run: 'click',
         },
         tourUtils.goToCheckout(),
+        {
+            trigger: "form[data-company-country-code=BR]",
+            async run() {
+                //Wait the form is loaded before continue. If not, there is a problem
+                //whith _selectState()
+                await new Promise((r) => setTimeout(r, 1000));
```

Comment:
Not sure if it's possible but isn't there any element we could `waitFor` instead?